tests/ztest/mock/src/main.c

```diff
@@ -78,4 +80,5 @@ void test_main(void)
 			 );
 
 	ztest_run_test_suite(mock_framework_tests);
+	test_suite_fff();
```

Comment:
This appears messy to me.
Are you trying to test two different features (mock frameworks) inside the same test suite ?

Why not create a completely new test suite under: ` tests/ztest/mock_fff/`
or similar ?

Reply:
ACK, will split it into a separate suite, I wanted to put it in mock for discoverability.

Reply:
Done, but I'm getting an error now for `native_posix` (will post as a separate comment)